spring-cloud-gcp-autoconfigure/pom.xml

```diff
@@ -22,20 +22,112 @@
             <artifactId>spring-cloud-gcp-core</artifactId>
             <optional>true</optional>
         </dependency>
-        <!-- Configuration processor -->
+
+        <!-- Global -->
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-configuration-processor</artifactId>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>commons-logging</groupId>
+            <artifactId>commons-logging</artifactId>
```

Comment:
Where's this used?

Reply:
`GcpContextAutoConfiguration` uses `Log`, for example.

Reply:
I see, OK.